tox.ini

```diff
@@ -9,7 +9,7 @@ exclude = .git,.tox,migrations
 max-line-length = 120
 
 [pydocstyle]
-ignore = D200,D203,D212
+ignore = D200,D203,D212,D406,D407
```

Comment:
i think we should keep linting configurations in setup.cfg that way tox.ini will be clean.

Reply:
I originally preferred `tox.ini` since some packages didn't have a `setup.cfg`, but now most repos at least have a short `[bdist_wheel]` entry in it.